packages/knip/test/plugins/node-modules-inspector.test.ts

```diff
@@ -0,0 +1,22 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { main } from '../../src/index.js';
+import { resolve } from '../../src/util/path.js';
+import baseArguments from '../helpers/baseArguments.js';
+import baseCounters from '../helpers/baseCounters.js';
+
+const cwd = resolve('fixtures/plugins/node-modules-inspector');
+
+test('Find dependencies with the node-modules-inspector plugin', async () => {
+  const { counters } = await main({
+    ...baseArguments,
+    cwd,
+  });
+
+  assert.deepEqual(counters, {
+    ...baseCounters,
+    binaries: 1,
```

Comment:
Sorry I missed this initially: to go full-circle and have everything covered, please add `node_modules/node-modules-inspector/package.json` with a `bin` field. Then when Knip sees e.g. `node-modules-inspector build` we'll no longer have this "unlisted binary" reported.